tests/queries_ported/0_stateless/02125_lz4_compression_bug.sh

```diff
@@ -9,12 +9,12 @@ for format in Native Values JSONCompactEachRow TSKV TSV CSV JSONEachRow JSONComp
 do
     echo $format
     ${CLICKHOUSE_CLIENT} --query "DROP STREAM IF EXISTS file"
-    ${CLICKHOUSE_CLIENT} --query "CREATE STREAM file (x uint64) ENGINE = File($format, '${CLICKHOUSE_DATABASE}/data.$format.lz4')"
+    ${CLICKHOUSE_CLIENT} --query "CREATE STREAM file (x uint64) ENGINE = File($format, '${CLICKHOUSE_DATABASE}/data.$format.lz4') SETTINGS query_mode='table'"
```

Comment:
incorrect modification, I will debug this tomorrow.
this script the whole result has a major difference from the expected one.
for local testing can use 
```bash
#!/usr/bin/env bash
# Tags: no-parallel, no-fasttest

for format in Native Values JSONCompactEachRow TSKV TSV CSV JSONEachRow JSONCompactEachRow JSONStringsEachRow
do
    echo $format
    ./proton-client --query "DROP STREAM IF EXISTS file"
    ./proton-client --query "CREATE STREAM file (x uint64) ENGINE = File($format, 'default/data.$format.lz4')"
    for size in 10000 100000 1000000 2500000
    do
        ./proton-client --query "TRUNCATE STREAM file"
        ./proton-client --query "INSERT INTO file SELECT * FROM numbers($size)"
        ./proton-client --query "SELECT max(x) FROM file"
    done
done

./proton-client --query "DROP STREAM file"

```